SRC/NBASIC/nbas111e.txt

```diff
@@ -796,26 +791,26 @@ ensemble use of NestorBASIC and InterNestor Suite.
 
 9.4. USING INTERNESTOR LITE
 
-InterNestor Lite is much simpler than InterNestor Suite, hence using it from 
-NestorBASIC is also a simpler task. Only one function is provided (function 
-86), allowing the execution of any routine of the InterNestor Lite code 
-segment. To read or write in its data segment, you must use the GET_VAR, 
-SET_VAR and COPY_DATA routines of the code segment.
+InterNestor Lite is much simpler than InterNestor Suite, thus it's also easier
+to use it from within the NestorBASIC. Only one function is provided (function 

```

Comment:
"the NestorBASIC" -> just "NestorBASIC".

Reply:
Ok.